packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -869,12 +870,15 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
     super.settings,
     super.requestFocus,
     this.popUpAnimationStyle,
-  }) : itemSizes = List<Size?>.filled(items.length, null),
+  }) : assert(position != null || positionBuilder != null,
```

Comment:
The assert here should be changed as well.

Reply:
done